poc_mobile_verifier/src/heartbeats.rs

```diff
@@ -29,7 +29,15 @@ pub struct HeartbeatKey {
 #[derive(Default)]
 pub struct HeartbeatValue {
     pub reward_weight: Decimal,
-    pub timestamps: Vec<NaiveDateTime>,
+    pub hours_seen: [bool; 24],
+}
+
+impl HeartbeatValue {
+    fn heartbeat_count(&self) -> usize {
+        self.hours_seen
+            .iter()
+            .fold(0, |hbs, seen| *seen as usize + hbs)
```

Comment:
casting a `boolean` to a numeric type seems very anti-rust, even if you're doing it explicitly; just when i thought i had you figured out rust...

Reply:
It's quite rusty! https://doc.rust-lang.org/std/primitive.bool.html